mysql-test/t/ps-3958.test

```diff
@@ -0,0 +1,18 @@
+#
```

Comment:
On a second thought, should this be added to the main --innodb-temp-tablespace-encrypt testcase?